### PR TITLE
Add missing eol anchors

### PR DESCRIPTION
**Pull request by jacobilsoe**

### CODE REVIEW

File: all-the-icons.el [hunk 1]
```diff
@@ -142,19 +142,19 @@
...
+    ("\\.node$"         all-the-icons-alltheicon "nodejs"               :height 1.0  :face all-the-icons-green)
     ("\\.babelrc$"      all-the-icons-fileicon "babel"                  :face all-the-icons-yellow)
     ("\\.bashrc$"       all-the-icons-alltheicon "script"               :height 0.9  :face all-the-icons-dpink)
     ("\\.bowerrc$"      all-the-icons-alltheicon "bower"                :height 1.0 :v-adjust 0.0 :face all-the-icons-silver)
     ("^bower.json$"     all-the-icons-alltheicon "bower"                :height 1.0 :v-adjust 0.0 :face all-the-icons-lorange)
     ("\\.ini$"          all-the-icons-octicon "settings"                :v-adjust 0.0 :face all-the-icons-yellow)
-    ("\\.eslintignore"  all-the-icons-fileicon "eslint"                 :height 0.9  :face all-the-icons-purple)
-    ("\\.eslint"        all-the-icons-fileicon "eslint"                 :height 0.9  :face all-the-icons-lpurple)
+    ("\\.eslintignore$" all-the-icons-fileicon "eslint"                 :height 0.9  :face all-the-icons-purple)
+    ("\\.eslint$"       all-the-icons-fileicon "eslint"                 :height 0.9  :face all-the-icons-lpurple)
```
Comment (wyuenho) [inline all-the-icons.el:157]: pretty sure this isn't a mistake. `.eslint` can have multiple extensions.

**Review by wyuenho — CHANGES_REQUESTED** [review]

File: all-the-icons.el (hunk 1 above)
Comment (jacobilsoe) [reply all-the-icons.el]: I reverted that change

**Comment by wyuenho** [comment]
There are still a couple like `.git` that don't have eol anchors, do you intend to leave them?

**Comment by jacobilsoe** [comment]
I only changed those I was sure of. .git for example also matches  .gitignore. So maybe it should be .gitignore$ instead or have two?

**Comment by wyuenho** [comment]
Thanks!

**Pull request merged**